corehq/apps/accounting/migrations/0033_merge.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('accounting', '0032_merge'),
+        ('accounting', '0031_merge'),
+    ]
+
+    operations = [
+    ]
```

Comment:
these accounting migrations don't seem to belong here. Perhaps from a bad merge?